### PR TITLE
Soe 401

### PR DESCRIPTION
**Pull request by boznik**
minor edits. still not sure if this is right...

### CODE REVIEW

File: includes/JumpstartSitesEngineering.php [hunk 1]
```diff
@@ -664,5 +672,82 @@ private function fetch_jse_content_beans($endpoint) {
...
+    $sunet = strtolower(trim($sunetid));
+    $authname = $sunet . '@stanford.edu';
+
+    $sunet_role = user_role_load_by_name('SUNet User');
+    $owner_role = user_role_load_by_name('site owner');
+    $editor_role = user_role_load_by_name('editor');
+    $admin_role = user_role_load_by_name('administrator');
+    $member_role = user_role_load_by_name('site member');
+
+    // Change user 1, currently admin, to jse-admins
```
Comment (jbickar) [inline includes/JumpstartSitesEngineering.php:714]: You're not changing the user name here, only the email address. That may be what you want to do, I'm not sure.

```
% drush uinf 1


 User ID       :  1                             
 User name     :  admin                         
 User mail     :  jse-admins@lists.stanford.edu 
 User roles    :  authenticated user            
                  SUNet User                    
                  administrator                 
 User status   :  active           
```

File: includes/JumpstartSitesEngineering.php [hunk 2]
```diff
@@ -664,5 +672,82 @@ private function fetch_jse_content_beans($endpoint) {
...
+    require_once DRUPAL_ROOT . '/includes/password.inc';
+    $install_vars = variable_get('stanford_jumpstart_install', array());
+    $config_form_data = $install_state['forms']['install_configure_form'];
+
+    // Get some stored variables.
+    if ($install_state['interactive']) {
+      $full_name  = isset($install_vars['full_name']) ? $install_vars['full_name'] : "School of Engineering";
+      $sunetid    = isset($install_vars['sunetid']) ? $install_vars['sunetid'] : 'jse-admins';
+    }
+    else if (function_exists('drush_get_option')) {
```
Comment (jbickar) [inline includes/JumpstartSitesEngineering.php:694]: `elseif`

**Comment by jbickar** [comment]
When I install with the following command:
`drush si stanford_sites_jumpstart_engineering -y install_configure_form.stanford_sites_requester_email="jbickar@stanford.edu" install_configure_form.stanford_sites_requester_name="John Bickar" install_configure_form.stanford_sites_requester_sunetid="jbickar"`

here's what happens:
1. User 1's email is set to "jse-admins@lists.stanford.edu" and name is "admin" (good)
2. The `soe:jse-admins` workgroup gets mapped to the "administrator" role (also good)
3. The requester (me, jbickar@stanford.edu) does not get an account created, and therefore does not get the "site owner" role mapped to its account (not so good)

Testing one more pass with additional parameters to `drush si`...

**Comment by jbickar** [comment]
I think the reason why this isn't creating the WMD user for me on local is because it's looking at `stanford.profile` to do that, and that code is checking to see if the hosting is on Stanford Sites:

https://github.com/SU-SWS/Stanford-Drupal-Profile/blob/7.x-2.x/stanford.profile#L139

So we need to test on Sites. Do you have a build site we can use?

**Comment by boznik** [comment]
yeah, there's a build site at:
https://sites.stanford.edu/jse-dev/

**Comment by jbickar** [comment]
Merging for testing...

**Pull request merged**